server/lib/models/matchmaking-times.ts

```diff
@@ -154,6 +154,51 @@ export async function getPastMatchmakingTimes(
   }
 }
 
+/**
+ * Get matchmaking schedule. The schedule represents future N (where N defaults to 2) matchmaking
```

Comment:
Nit: ideally the first sentence of a method comment wouldn't be exactly the same as the method name :)

`Retrieve the schedule of currently applicable matchmaking status changes.` would be a slight improvement